joomla/joomla-cms
# Cleanup Installer plugins

Pull Request for Issue # .

### Summary of Changes
🤪 Just some small cleanup... 


### Testing Instructions
Check if the extension is installed correctly using the drag and drop installer.


### Expected result
The installer is working correctly.


### Actual result
The installer is working correctly.


### Documentation Changes Required
No
